mention-bot.js

```diff
@@ -9,7 +9,17 @@
  * @flow
  */
 
-var downloadFileSync = require('download-file-sync');
+var githubWebClient = require('./githubWebClient');
+
+var downloadFileSync = function(url, cookies) {
+  var command = 'curl -L ' + url;
+  if(cookies) {
+    command += ' -H ' + '"Cookie: ' + cookies + '"';
+  }
+
+  return require('child_process')
+    .execSync(command).toString();
```

Comment:
can you use `execSync(command, {encoding: 'utf8'})` instead of `execSync(command).toString()`. This avoids creating a large buffer that's just going to be turned into a string right after